runtime/protocol/http-client/common/src/aws/smithy/kotlin/runtime/http/operation/OperationTelemetry.kt

```diff
@@ -79,5 +72,12 @@ internal fun<I, O> SdkHttpOperation<I, O>.instrument(): Pair<TraceSpan, Coroutin
         telemetry.spanKind,
         parentCtx,
     )
+
+    val telemetryCtx = TelemetryProviderContext(telemetry.provider) +
+        LoggingContextElement(
+            "rpc" to rpcName,
+            "sdkInvocationId" to context.sdkInvocationId,
+        )
```

Comment:
**Question**: In the previous code (deleted above), this new `telemetryCtx` also included `TelemetryContextElement(parentCtx)`. Why is that removed now?

Reply:
It's now set in the coroutine [extensions](https://github.com/awslabs/smithy-kotlin/blob/562508c799ffff7e257c6d6818fd3a91c22efcb3/runtime/observability/telemetry-api/common/src/aws/smithy/kotlin/runtime/telemetry/trace/CoroutineContextTraceExt.kt#L70-L71) so that it gets set at the correct place/time by default.